api/registrations/views.py

```diff
@@ -96,10 +109,27 @@ def get_default_odm_query(self):
         query = base_query & permission_query
         return query
 
+    def is_blacklisted(self, query):
+        for query_param in query.nodes:
```

Comment:
I don't know why it's called nodes. But if I loop through query.nodes I am looping through the various query_params, such as is_deleted is True.  Some of them will be a QueryGroup like is_public, eq, True or contributors, icontains, rytg8, but I'm not concerned with those.  

Anyways, if a query_param has an attribute on it, it will be a query_param the user passed in the URL,  like filtering on title, or public.not one that we put on it behind the scenes.  If I am filtering on public, the query_param will have an attribute `is_public`.